docs/features/logforwarding/outputs/splunk-forwarding.adoc

```diff
@@ -30,11 +31,15 @@ spec:
       type: splunk
       splunk:
         authentication:
-          token:
+          token: # <1>
             key: hecToken
             secretName: splunk-secret
-        url: 'http://example-splunk-hec-service:8088'
-        index: '{.log_type||"undefined"}'
+        url: 'http://example-splunk-hec-service:8088' # <2>
+        index: '{.log_type||"undefined"}' # <3>
```

Comment:
We talked about adding `payloadKey` like syslog to allow setting the splunk `event` to something else.  This would imply the `sourceType` is dependent upon the payload.   Should we do this?

Reply:
Don't see the reason todo it, also Vector not support such configuration parameter

Reply:
Vector does support it by way of setting "message" to whatever value you want.  It will map that to the 'event'.  I'll defer in pushing on payload key mostly because I'm not certain we need to allow changing it.  The feature does not request it.  I'll defer to other opinions and input if there is a larger need